usaspending_api/database_scripts/etl/transaction_delta_view.sql

```diff
@@ -4,12 +4,12 @@ DROP VIEW IF EXISTS transaction_delta_view;
 CREATE VIEW transaction_delta_view AS
 SELECT
   UTM.transaction_id,
-  FPDS.detached_award_proc_unique,
-  FABS.afa_generated_unique,
+  UTM.detached_award_proc_unique,
+  UTM.afa_generated_unique,
 
   CASE
-    WHEN FPDS.detached_award_proc_unique IS NOT NULL THEN 'CONT_TX_' || UPPER(FPDS.detached_award_proc_unique)
-    WHEN FABS.afa_generated_unique IS NOT NULL THEN 'ASST_TX_' || UPPER(FABS.afa_generated_unique)
+    WHEN UTM.detached_award_proc_unique IS NOT NULL THEN 'CONT_TX_' || UPPER(UTM.detached_award_proc_unique)
+    WHEN UTM.afa_generated_unique IS NOT NULL THEN 'ASST_TX_' || UPPER(UTM.afa_generated_unique)
```

Comment:
since these are now coming from the UTM, can we just ensure that the UTM always projects them with UPPERCASE, to save on having to uppercase here?

It also helps with not missing indexes, since if an index exists on `UTM.detached_award_proc_unique` or `UTM.afa_generated_unique`, it would likely exist on the non-UPPER form of those fields (though doesn't appear to be an index on either of those).

Reply:
Not sure how an index would be useful for these string concatenations. A quick check didn't show any lower-case letters in the fields, so I'll remove the upper functions for now